src/packages/tapisetl/controllers/Index.py

```diff
@@ -1,6 +1,12 @@
 from packages.tapisetl.ETLController import ETLController
+from utils.Prompt import prompt
 
 
 class Index(ETLController):
+    directories = {}
```

Comment:
Remove. Doesn't look like there is any use for this attribute. Also, I generally don't use class attributes for anything.